src/commands/pay/me/processor.ts

```diff
@@ -76,6 +76,19 @@ export async function run({
       hasTarget ? "" : "Please copy the message below and send to your friend"
     }`,
   })
+  if (platform == "discord") {
```

Comment:
no need change this

Reply:
<img width="861" alt="Screen Shot 2023-03-28 at 7 08 24 PM" src="https://user-images.githubusercontent.com/39359294/228231302-cdf45018-39b9-44e6-ba36-a51a555504ca.png">
I need this line